.github/workflows/github-action-build.yml

```diff
@@ -119,6 +119,10 @@ jobs:
         run: psql -h localhost -U postgres -v DB_NAME=texera_db_for_test_cases -f core/scripts/sql/texera_ddl.sql
         env:
           PGPASSWORD: postgres
+      - name: Create texera_db_for_test_cases
```

Comment:
This line is duplicated due to merge conflicts.

Reply:
Good catch, fixed.